File: leetcode-python-go/meituan-001. 小美的用户名/main.go
package main

import (
	"fmt"
	"unicode"
)

func check(s string) string {
	d := false
	for i, c := range s {
		if i == 0 {
			if !unicode.IsLetter(c) {
				return "Wrong"
			}
		}
		if !unicode.IsLetter(c) && !unicode.IsNumber(c) {
			return "Wrong"
		} else if unicode.IsNumber(c) {
			d = true
		}
	}
	if d {
		return "Accept"
	} else {
		return "Wrong"
	}
}

func main() {
	var T int
	var s string
	fmt.Scanln(&T)
	for i := 0; i < T; i++ {
		fmt.Scanln(&s)
		fmt.Printf("%s\n", check(s))
	}
}

File: leetcode-python-go/meituan-001. 小美的用户名/main.py
nums = int(input())
for _ in range(nums):
    ret, li = True, [False, False]
    strs = input()
    if not (strs[0].islower() or strs[0].isupper()):
        ret = False

    for i in strs:
        if i.islower() or i.isupper():
            li[0] = True
        elif i.isdigit():
            li[1] = True
        else:
            ret = False
            break
    if not all(li):
        ret = False
    print("Accept" if ret else "Wrong")
